src/components/Collapsible/test.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Collapsible from './index';
+import {mount, shallow} from 'enzyme';
+
+
+describe('Collapsible', () => {
+
+    it('renders correctly', () => {
+        const tree = renderer
+            .create(<Collapsible anchor={(<div>Testing</div>)}>Test</Collapsible>)
+            .toJSON();
+        expect(tree).toMatchSnapshot();
+    });
+
+    it('mounts correctly', () => {
+        const collapsible = shallow(<Collapsible anchor={(<div>Testing</div>)}>Test</Collapsible>);
+        expect(collapsible.hasClass('brew-Collapsible')).toBe(true);
+    });
+
+    it('correctly adds a className', () => {
+        const className = `class-${Math.floor(Math.random() * 1000)}`;
+        const collapsible = shallow(<Collapsible anchor={(<div>Testing</div>)} className={className}>Test</Collapsible>);
+        expect(collapsible.hasClass(className)).toBe(true);
+    });
+
+    it('correctly adds open class when isCollapsed is false', () => {
+        const collapsible = mount(<Collapsible anchor={(<div>Testing</div>)} isCollapsed={false}>Test</Collapsible>);
+        expect(collapsible.prop('isCollapsed')).toBe(false);
+    });
+
+});
```

Comment:
We probably would want another test to make sure children is being properly passed through, and one that tests to make sure the anchor content is passing through correctly.

Reply:
AnchorContent is required, so wouldn't the tests to check if component is mounted fail if anchorContent was not passed correctly? Or do I need a specific test for that

Reply:
Even if anchor content is required that doesn't guarantee that the component is displaying that anchor content, so it would still need another test.

Reply:
Most of the props should be able to be verified if you just add them to your snapshot.